libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewStyles.kt

```diff
@@ -36,14 +40,24 @@ internal class NavigationViewStyles(context: Context) {
         MutableStateFlow(ViewStyleCustomization.defaultSpeedLimitStyle())
     private var _speedLimitTextAppearance: MutableStateFlow<Int> =
         MutableStateFlow(ViewStyleCustomization.defaultSpeedLimitTextAppearance())
-    private var _destinationMarker: MutableStateFlow<Int> =
-        MutableStateFlow(ViewStyleCustomization.defaultDestinationMarker())
     private var _roadNameBackground: MutableStateFlow<Int> =
         MutableStateFlow(ViewStyleCustomization.defaultRoadNameBackground())
     private var _roadNameTextAppearance: MutableStateFlow<Int> =
         MutableStateFlow(ViewStyleCustomization.defaultRoadNameTextAppearance())
     private var _maneuverViewOptions: MutableStateFlow<ManeuverViewOptions> =
         MutableStateFlow(ViewStyleCustomization.defaultManeuverViewOptions())
+    private var _destinationMarkerAnnotationOptions: MutableStateFlow<PointAnnotationOptions> =
+        MutableStateFlow(
+            ViewStyleCustomization
+                .defaultMarkerAnnotationOptions()
+                .withIconImage(
+                    ContextCompat.getDrawable(
+                        context,
+                        R.drawable.mapbox_ic_destination_marker
+                    )!!.toBitmap()
+                )
+                .withIconAnchor(IconAnchor.BOTTOM)
```

Comment:
Should this be moved to `defaultMarkerAnnotationOptions` (which in turn renamed to `defaultDestinationMarkerAnnotationOptions`)?

Reply:
I can move `withIconAnchor` to `ViewStyleCustomization.defaultDestinationMarkerOptions` but `withIvonImage` would have to stay here because it needs `context`. I prefer to have all properties live in one place than having some in one file vs rest in another.

Reply:
I think you can add `Context` parameter to `ViewStyleCustomization.defaultDestinationMarkerAnnotationOptions`. Like `ViewStyleCustomization.defaultInfoPanelPeekHeight` has one.

Reply:
The problem with the current implementation is that you can't simply do `destinationMarkerAnnotationOptions = defaultMarkerAnnotationOptions()` to reset the overridden options, like javadoc suggests.